src/beast/extras/beast/unit_test/reporter.hpp

```diff
@@ -283,6 +283,7 @@ reporter<_>::
 on_log(std::string const& s)
 {
     os_ << s;
+    os_.flush();
```

Comment:
I don't think there's anything wrong with a local patch while we wait for upstream, especially if this is the only thing that fixes our issue.